### DISCUSSION TITLE
Need touch sampling settings as well.

### DISCUSSION THREAD

**Discussion by thedeepsinghal** [tribalfs/GalaxyMaxHzPub#238] [Ideas]
Mid range and most new samsung phone having high touch sampling only while games running . need it to adjust in normal apps also

**Comment by tribalfs** [comment]
Not possible.

**Reply by thedeepsinghal** [reply]
Oh! Damn is it so? Why it cant be done? If they can enable high touch sampling rate in games but not in apps?

**Reply by tribalfs** [reply]
Samsung/Google can but not the app.
(It's probably to done to improve battery usage)

**Reply by thedeepsinghal** [reply]
How that high sampling rate trigger in games?. What kind of service or event it is which makes phone to act fast. Cant we enable that service via app?  As i saw people modded touch sampling using magisk module mods and adb commands so if we can get toggle for samsung it will be great. Will depend on user if turn on then battery will be affected.

**Reply by tribalfs** [reply]
I honestly don't know how exactly.  I haven't dug to this stuff.  Maybe will check when I have time.

**Reply by thedeepsinghal** [reply]
Please check thanks 🙏
